concent_api/core/transfer_operations.py

```diff
@@ -119,11 +119,13 @@ def create_file_transfer_token(
             3 * settings.CONCENT_MESSAGING_TIME +
             2 * settings.MAXIMUM_DOWNLOAD_TIME
         )
-    if operation == 'download':
+    elif operation == 'download':
         token_expiration_deadline = (
             report_computed_task.task_to_compute.compute_task_def['deadline'] +
             settings.SUBTASK_VERIFICATION_TIME
         )
+    else:
```

Comment:
This code will never be reach (there is assert above), I will remove it.

Reply:
Isn't it an error, at least in pylint. That we have `if` than `elif` without `else`?

Reply:
No, at least it doesn't show any warnings when I was running it.